pkg/kubelet/metrics/metrics.go

```diff
@@ -41,6 +41,7 @@ const (
 	PodWorkerDurationKey               = "pod_worker_duration_seconds"
 	PodStartDurationKey                = "pod_start_duration_seconds"
 	PodStartSLIDurationKey             = "pod_start_sli_duration_seconds"
+	PodStartE2EDurationKey             = "pod_start_e2e_duration_seconds"
```

Comment:
nit for naming: how about `pod_start_total_duration_seconds`?

Reply:
e2e in kubernetes has a hard bias towards e2e tests, I suggest other name less confusing

Reply:
makes sense. I changed it to pod_start_total_duration_seconds as Qiutong suggests.